### PR TITLE
Remove unused python dependencies

### PR DESCRIPTION
**Pull request by Bomme**
**Issue(s)**
as part of #1083 

**Description**
Remove dependencies that are currently not used (and outdated) 

**Deployment steps**:
None

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -47,10 +45,8 @@ pyparsing==2.4.7
 pysolr==3.10.0b1
 pysndfile==1.4.3
 python-cjson==1.1.0
-python-memcached==1.53
 pytz==2020.1
 PyYAML==3.11
-raven==6.10.0
```
Comment (ffont) [inline requirements.txt:53]: I think we need this for sentry reports right? @alastair

Comment (Bomme) [reply requirements.txt:53]: Raven is deprecated in favor of Sentry-Python (`sentry-sdk`)

Comment (ffont) [reply requirements.txt:53]: ah right, I see, I forgot that we made that change :)

**Comment by ffont** [comment]
Thanks @Bomme, merged!

**Pull request merged**